### ISSUE TITLE
stc.plot() throws warning by default

### ISSUE THREAD

**Issue by mainakjas** [mne-tools/mne-python#2093]
If I don't set any `clim` or `fmin` etc for that matter -- just a plain `stc.plot()`, it throws a warning saying:

```py
mne/viz/_3d.py:554: DeprecationWarning: Using fmin, fmid, fmax is deprecated and will be removed in v0.10. Use "clim" instead.
  DeprecationWarning)
```

This is not desired behaviour.

cc @wronk

**Comment by Eric89GXL** [comment]
I think we might need it to be that way for one release to change the
behavior to use percentiles.

**Comment by wronk** [comment]
Before, stc.plot() would put in default fmin, fmid, fmax values, which we need to deprecate. I guess in the future we should have it implicitly use default percentiles with the `mne` colormap, but we need at least one release in between, correct?

@Eric89GXL, lmk if you want me to change anything though.

**Comment by Eric89GXL** [comment]
I guess of all entries are none, we should use old behavior and give a
warning that it will change to clim auto next release.

**Comment by wronk** [comment]
Sure, I'll open a PR in a few.

**Comment by mainakjas** [comment]
I guess the warning is confusing for a new user who hasn't even started exploring these different `clim` options and is just looking to use the default. Now, he has to deal with 4 new parameters

**Comment by Eric89GXL** [comment]
It's only one new parameter (and three old ones), they should use `clim='auto'` as suggested by what will be the new warning. Is that clear enough?

**Comment by mainakjas** [comment]
okay for that as long as it's clear to the user that they don't need to do anything with the warning

**Comment by Eric89GXL** [comment]
well... it will be the same as with any deprecation warning, in the sense that if they do nothing, the behavior might change on the next release

**Comment by wronk** [comment]
See Clim fix #2094
